### PR TITLE
HDDS-10492. [Doc] Update zh translation to Recon Architecture

### PR DESCRIPTION
**Pull request by aierate**
## What changes were proposed in this pull request?

Update Chinese translation to Recon Architecture to match English . The updated filename is ReconApi.zh.md. 

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-10492

## How was this patch tested?

It‘s only Markdown file, don't test.

### CODE REVIEW

File: hadoop-hdds/docs/content/interface/ReconApi.zh.md [hunk 1]
```diff
@@ -251,6 +741,114 @@ Recon API v1 是一组 HTTP 端点，可以帮助您了解 Ozone 集群的当前
...
+
+**参数**
+
+* volume (可选)
+
+  卷以字符串形式表示，不包含任何协议前缀。
+
+* prevKey (可选)
+
+  如果未指定卷，则忽略 prevKey，并且仅返回给定 prevKey 之后的存储桶。
```
Comment (ChenSammi) [inline hadoop-hdds/docs/content/interface/ReconApi.zh.md:806]: 如果未指定卷，则忽略 prevKey，并且仅返回给定 prevKey 之后的存储桶。 -> 

返回给定 prevKey 之后的存储桶。 如果未指定卷，则忽略 prevKey。

File: hadoop-hdds/docs/content/interface/ReconApi.zh.md [hunk 2]
```diff
@@ -222,7 +222,497 @@ Recon API v1 是一组 HTTP 端点，可以帮助您了解 Ozone 集群的当前
...
+
+**参数**
+
+* path
+
+  字符串形式的路径请求，不包含任何协议前缀。
+
+**回传**
+
+返回路径的基本摘要，包括实体类型和路径下对象的聚合计数。
```
Comment (ChenSammi) [inline hadoop-hdds/docs/content/interface/ReconApi.zh.md:519]: 摘要 -》 信息汇总

File: hadoop-hdds/docs/content/interface/ReconApi.zh.md [hunk 3]
```diff
@@ -222,7 +222,497 @@ Recon API v1 是一组 HTTP 端点，可以帮助您了解 Ozone 集群的当前
...
+
+  一个布尔值，默认值为 `false`。如果设置为 `true`，则会计算路径下键的磁盘使用情况。
+
+* replica (可选)
+
+  一个布尔值，默认为 `false`。如果设置为 `true`，则会计算键的副本大小的磁盘使用情况。
+
+**回传**
+
+返回路径下所有子路径的磁盘使用情况。规范化 `path` 字段，直接返回路径下键的总大小作为
```
Comment (ChenSammi) [inline hadoop-hdds/docs/content/interface/ReconApi.zh.md:603]: 直接返回路径下键的总大小 -》 返回路径下直接健的总大小

**Comment by aierate** [comment]
Thank you, @ChenSammi  I have corrected the errors in the Chinese description based on your suggestions.

**Review by ChenSammi — APPROVED** [review]
Thanks @aierate for the contribution.

**Pull request merged**